spec/select_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'select.rb'
+
+describe Select do
+		let(:burger){double (:burger)}
+		it 'shows the menu' do 
+			men = {'Burger'=> 10, 'Pint'=> 4.50,'Steak'=>18,'Wine'=> 22,'Chips'=> 3.25}
+			expect(subject.menu).to eq men
+		end
+		it 'shows the menu' do 
+			men = {'Burger'=> 10, 'Pint'=> 4.50,'Steak'=>18,'Wine'=> 22,'Chips'=> 3.25}
+			expect(subject.see).to eq men
+		end
+
+		it 'has an order' do
+ 			expect(subject).to respond_to(:order)
+		end
+		it 'has an due amount' do
+ 			expect(subject).to respond_to(:order)
+ 		end
+
+		describe 'pick' do 
+			it 'take an argument for dish and argument for quantity' do 
+				allow(subject).to receive(:pick).with(2).and_return(2)
+				expect(subject.pick(2)).to eq(2)
```

Comment:
This allow might be creating a vacuous test
-- https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#tests-should-test-real-behaviours-not-stubs